elements/rh-navigation-primary/rh-navigation-primary-item.ts

```diff
@@ -0,0 +1,130 @@
+import type { IconNameFor, IconSetName } from '@rhds/icons';
+
+import { LitElement, html, isServer } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { state } from 'lit/decorators/state.js';
+import { query } from 'lit/decorators/query.js';
+import { property } from 'lit/decorators/property.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+import { consume } from '@lit/context';
+import { context } from './context.js';
+
+import { InternalsController } from '@patternfly/pfe-core/controllers/internals-controller.js';
+
+import { colorPalettes, type ColorPalette } from '@rhds/elements/lib/color-palettes.js';
+import { themable } from '@rhds/elements/lib/themable.js';
+
+import '@rhds/elements/rh-icon/rh-icon.js';
+
+import './rh-navigation-primary-item-menu.js';
+
+import styles from './rh-navigation-primary-item.css';
+
+@customElement('rh-navigation-primary-item')
+@colorPalettes
```

Comment:
is it necessary for this to have a color-palette attr?

Reply:
Maybe not anymore.  Might be an orphan now. I will investigate.